### PR TITLE
Task-48693: Agenda event planned jitsi call isn't launched when space name is edited afterwards

### PR DESCRIPTION
**Pull request by AzmiTouil**
Prior to this fix, we couldn't lunch a scheduled jitsi call in a space event after renaming the space.
To fix this problem we need to use the space identity id as ownerId instead of the spacePrettyName

### CODE REVIEW

File: services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java [hunk 1]
```diff
@@ -238,9 +238,10 @@ public String getType() {
      * Instantiates a new space event info.
      *
      * @param socialSpace the social space
+     * @param spaceIdentityId the space identity id
      */
-    public SpaceEventInfo(Space socialSpace) {
-      super(socialSpace.getPrettyName(), socialSpace.getDisplayName());
+    public SpaceEventInfo(Space socialSpace, String spaceIdentityId) {
```
Comment (ahamdi) [inline services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java]: Why do we need to keep both ? could we remove socialSpace attribute ?

Comment (AzmiTouil) [reply services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java]: IMO we can't remove it, because we need the space group id.

Comment (ahamdi) [reply services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java]: We cna get the groupID by getting the space with its identity  ID

File: services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java [hunk 2]
```diff
@@ -237,12 +237,16 @@ public String getType() {
...
-    public SpaceEventInfo(Space socialSpace, String spaceIdentityId) {
-      super(spaceIdentityId, socialSpace.getDisplayName());
-      this.groupId = socialSpace.getGroupId();
+    public SpaceEventInfo(Identity spaceIdentity) {
+      super(spaceIdentity.getId(), spaceIdentity.getRemoteId());
+      Space space = spaceService.getSpaceByPrettyName(spaceIdentity.getRemoteId());
+      this.profileLink = space.getUrl();
+      String avatar = space.getAvatarUrl();
+      avatar = avatar != null ? avatar : LinkProvider.SPACE_DEFAULT_AVATAR_URL;
+      this.avatarLink = avatar;
```
Comment (ahamdi) [inline services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java]: Those 3 lines could be combined in one

**Review by ahamdi — APPROVED** [review]

**Pull request merged**